processor/src/main/java/org/mobilitydata/gtfsvalidator/processor/GtfsAnnotationProcessor.java

```diff
@@ -48,6 +48,10 @@ public class GtfsAnnotationProcessor extends AbstractProcessor {
 
   private final Analyser analyser = new Analyser();
 
+  public GtfsAnnotationProcessor() {
```

Comment:
@bdferris-v2 FYI looks like some debug code may have snuck into this PR

Reply:
Fixed.   It's kind of a bummer that when the code processor fails, it seems to do so in very obscure ways :(

Reply:
💯 😅